src/backend/macos/utils/window.rs

```diff
@@ -10,8 +10,12 @@ pub trait INSWindow: INSObject {
     fn from_raw_window_handle(h: &RawWindowHandle) -> Id<Self> {
         match h {
             RawWindowHandle::AppKit(h) => {
-                let id = h.ns_window as *mut Self;
-                unsafe { Id::from_ptr(id) }
+                let id = h.ns_view.as_ptr() as *mut Self;
+                let id: Id<Self> = unsafe { Id::from_ptr(id) };
```

Comment:
Is using `*mut Self` for those right? 
`*mut Self` evaluates to `*mut NSWindow`
but the field contains `*mut NSView`

Reply:
Perhaps just do c_void (with a comment saying that it is a view) if you find dealing with `NSView` too cumbersome

Reply:
I used NSObject, which NSView inherits from, I think that's more correct?

Reply:
Yep, sounds good